### ISSUE TITLE
[Feature Request]: Enhance User Experience by Modifying Search Trigger Event in Secondary Header

### ISSUE THREAD

**Issue by fabiomlferreira** [rappasoft/laravel-livewire-tables#1404]
### Overview

Currently, the search functionality in the secondary header triggers the search upon the blur event. This behavior might not be as intuitive or user-friendly as it could be.

**Proposed Solution**
I propose adjusting the default event to initiate the search to be the pressing of the Enter key, with an optional debounce of 1 second.

**Example & Additional Context**
A great implementation example of this can be seen in the Filament table builder. In their setup, the search typically triggers approximately 1-2 seconds after the user ceases typing, and also activates when the Enter key is pressed.

**Suggested Approach**
Adopting a similar approach across all input search fields by default would enhance the user experience. While the main search allows us to select this, the secondary header currently does not provide this option. Implementing a consistent search trigger across all search inputs—primary and secondary—would provide a more cohesive and user-friendly experience.

**Comment by lrljoe** [comment]
Yep, this is actually on the to-do list, along with being able to set custom attributes on each filter input.

I'll leave this open so that people are aware that it's in progress, thanks for a very comprehensive request!
